package.json

```diff
@@ -20,12 +20,14 @@
   "sideEffects": false,
   "type": "module",
   "exports": {
-    "./protocols": "./src/protocols.json",
-    "./tokens/mainnet": "./src/tokens/mainnet.json",
+    "./protocols": "./output/protocols.ts",
```

Comment:
Why not `dist`? That's more standard I think

Reply:
yes, i did not want to change eslint config though. But since we don't need any of the ignores, i just updated it 👍